modules/addons/src/gltf/gltf-scenegraph-loader.js

```diff
@@ -1,7 +1,12 @@
+/* global window */
 import {GLTFParser} from '@loaders.gl/gltf';
 import {DracoDecoder} from '@loaders.gl/draco';
 import createGLTFObjects from './create-gltf-objects';
 
+function wait() {
+  return new Promise(resolve => window.setTimeout(resolve, 100));
```

Comment:
`requestAnimationFrame`. Avoid having code running out side an `rAF` callback, since there's a good chance it will include GL code.